MSMAll/scripts/MSMAll.sh

```diff
@@ -19,10 +19,10 @@ opts_SetScriptDescription "implements MSM-All Registration main script"
 #mandatory
 #general inputs
 opts_AddMandatory '--study-folder' 'StudyFolder' 'path' "folder that contains all subjects" '--path'
-opts_AddMandatory '--subject' 'Subject' '100206' "one subject ID"
+opts_AddMandatory '--session' 'Session' '100206' "one session ID" "--subject"
```

Comment:
I don't see any changes to the logic in this script, only a ton of variable replacements.  If this is the case, it would have been simpler to leave the variable name alone:
```suggestion
opts_AddMandatory '--session' 'Subject' '100206' "one session ID" "--subject"
<no changes to the rest of the script>
```
If there are changes in logic, please point them out.

Reply:
Yes, but the "new" variable naming conventions (session vs subject) are aligned with all other scripts. I think it is as good as a time as any to do this refactoring here as well.

Reply:
I see logic in both approaches but since I already did the renaming, will keep it for now